evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
     </div>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import participants' as title %}
+    {% blocktrans asvar question %}Do you really want to import the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
+    {% trans 'Import participants' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='importParticipantsModal' title=title question=question action_text=action_text btn_type='primary' %}
+    <script type="text/javascript">
+        function importParticipantsModalAction(dataId) {
+            var input = $("<input>")
+               .attr("type", "hidden")
+               .attr("name", "operation").val(dataId);
+            $('#participant-import-form').append(input);
+            $('#participant-import-form').submit();
+        };
+    </script>
+
+    {% trans 'Copy participants' as title %}
+    {% blocktrans asvar question %}Do you really want to copy participants? {% endblocktrans %}
+    {% trans 'Copy participants' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='copyParticipantsModal' title=title question=question action_text=action_text btn_type='primary' %}
+    <script type="text/javascript">
+        function copyParticipantsModalAction(dataId) {
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
+            $('#participant-copy-form').append(input);
+            $('#participant-copy-form').submit();  
+        };
+    </script>
+
+    {% trans 'Import contributors' as title %}
+    {% blocktrans asvar question %}Do you really want to import the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
+    {% trans 'Import contributors' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='importContributorsModal' title=title question=question action_text=action_text btn_type='primary' %}
+    <script type="text/javascript">
+        function importContributorsModalAction(dataId) {
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
+            $('#contributor-import-form').append(input);
+            $('#contributor-import-form').submit(); 
+        };
+    </script>
+
+    {% trans 'Copy contributors' as title %}
+    {% blocktrans asvar question %}Do you really want to copy contributors? {% endblocktrans %}
```

Comment:
please make it `Do you really want to copy the contributors?` (with `the` and without a trailing space).